### PR TITLE
Missing table fields

### PR DESCRIPTION
**Pull request by AlisonPAnderson**
I noticed several table fields were missing (including year built) - so I added them

### CODE REVIEW

File: backend/db/migrate/20220817011608_create_property_assessments.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@ def change
       t.string :bthrm_style3
       t.string :kitchen_type
       t.string :kitchen_style1
-      t.string :kitchen_style
+      t.string :kitchen_style  # is this kitchen_style2?
```
Comment (cheradenine) [inline backend/db/migrate/20220817011608_create_property_assessments.rb:54]: yes, should be kitchen_style2

File: backend/db/migrate/20220817011608_create_property_assessments.rb [hunk 2]
```diff
@@ -63,6 +63,17 @@ def change
       t.integer :kitchens_count
       t.integer :rooms_total_count
 
+    
+      # I noticed these fields were missing from the table.
```
Comment (cheradenine) [inline backend/db/migrate/20220817011608_create_property_assessments.rb:67]: Nice! You can remove the comment from the final code though.

**Review by cheradenine — COMMENTED** [review]
This PR has some other unrelated commits in it. Maybe you need to rebase or otherwise base this on main?
If it is ok I'm going to go into this same migration file and re-oder all the fields so that they are the same as the original dataset. And I can take care of adding these missing ones - or I can leave just those out if you want to do the honors?

**Comment by AlisonPAnderson** [comment]
closing per @cheradenine  - his request covers this issue!

**Pull request closed without merging**